packages/@aws-cdk/aws-servicediscovery/lib/instance.ts

```diff
@@ -0,0 +1,168 @@
+import cdk = require('@aws-cdk/cdk');
+import { NamespaceType } from './namespace';
+import { DnsRecordType, Service } from './service';
+import { CfnInstance } from './servicediscovery.generated';
+
+/**
+ * Properties to define ServiceDiscovery Instance
+ */
+export interface InstanceProps {
+  /**
+   * The instance attributes.
+   * FIXME: add defaults/validations
+   */
+  instanceAttributes: InstanceAttributes
+
+  /**
+   * The id of the instance resource
+   */
+  instanceId: string;
+
+  /**
+   * The Cloudmap service this resource is registered to.
+   */
+  service: Service;
+}
+
+/**
+ * Define a Service Discovery Instance
+ */
+export class Instance extends cdk.Construct {
```

Comment:
Missing `IInstance`, imports/exports, etc. Makes sure awslint rules are all enabled

Reply:
Cool -- I have them excluded in awslint already based on @rix0rrr's feedback.